ambry-api/src/main/java/com.github.ambry/network/NetworkSend.java

```diff
@@ -36,9 +36,9 @@
    */
   private final long sendStartTimeInMs;
 
-  private final NetworkRequestMetrics metrics;
+  private final NetworkRequestSend metrics;
 
-  public NetworkSend(String connectionId, Send payload, NetworkRequestMetrics metrics, Time time) {
+  public NetworkSend(String connectionId, Send payload, NetworkRequestSend metrics, Time time) {
```

Comment:
why not just send a metrics object like the others that we define and simply update the actual histogram required?